intro_to_R_tidyverse/01-intro_to_r.Rmd

```diff
@@ -8,14 +8,40 @@ output:
 
 **CCDL 2019**
 
-### This RMarkdown takes you through the basics of R  
+### This [R Markdown](http://rmarkdown.rstudio.com) Notebook takes you through the basics of R  
 
 *For more comprehensive tutorials:*  
   - For a tutorial on R, RStudio and R Markdown: https://ismayc.github.io/rbasics-book/  
-  - [Datacamp](https://www.datacamp.com/) has lots of good tutorials for R and the intro ones are free.       
   - For a lot of handy R cheatsheets: https://www.rstudio.com/resources/cheatsheets/  
   - For more guidance all the things you can do with R Notebooks:    
     https://bookdown.org/yihui/rmarkdown/notebook.html  
+
+#### A brief tutorial on how to use R Markdowns:
+
+When you execute code within the notebook, the results appear beneath the code. 
+Try executing this chunk by clicking the *Run* button within the chunk or by 
+placing your cursor inside it and pressing *Cmd+Shift+Enter*. 
+
+```{r}
+print("You ran a code chunk!")
+```
+
+Add a new chunk by clicking the *Insert Chunk* button on the toolbar or by 
+pressing *Cmd+Option+I* (Windows *Ctrl + Alt + I*).
+
+Try to put the following line into its own code chunk and run it:
+
+print("You made a code chunk!")
+
+#### Saving output from R Markdowns:
```

Comment:
I might move this section to right before session info. My rationale is that the inserting and especially _running_ chunks is essentially for following the rest of the notebook, but the information about saving can wait until the end. I am concerned about overwhelming someone with too much information at this stage when this is the first thing we go through.

Reply:
I was thinking about that originally, but didn't know if that would feel too choppy to break up the talk about R Markdowns.

Reply:
I think this sounds better to put this at the end. The concern about overwhelming people with information is a very real and valid concern.

Reply:
You could frame it more generally if you split it up. What I mean: have a little more information about the _why_/advantages of the notebook.

Reply:
If we move it to the end, is putting it before `About good coding practices` the move then? Or do we have to restructure the end part to include it?